sql/core/src/main/scala/org/apache/spark/sql/execution/SparkSqlParser.scala

```diff
@@ -373,7 +374,11 @@ class SparkSqlAstBuilder(conf: SQLConf) extends AstBuilder(conf) {
    * Create a [[DescribeQueryCommand]] logical command.
    */
   override def visitDescribeQuery(ctx: DescribeQueryContext): LogicalPlan = withOrigin(ctx) {
-    DescribeQueryCommand(visitQuery(ctx.query))
+    val query = ctx.query
+    val startIndex = query.start.getStartIndex
+    val endIndex = query.stop.getStopIndex
+    val interval = new Interval(startIndex, endIndex)
```

Comment:
nit. Can we remove the one-time used variables?
```scala
-    val startIndex = query.start.getStartIndex
-    val endIndex = query.stop.getStopIndex
-    val interval = new Interval(startIndex, endIndex)
+    val interval = new Interval(query.start.getStartIndex, query.stop.getStopIndex)
```

Reply:
@dongjoon-hyun Sure.